lib/webide-connections-monitor.js

```diff
@@ -0,0 +1,84 @@
+/* See license.txt for terms of usage */
+
+"use strict";
+
+module.metadata = {
+  "stability": "extermental"
+};
+
+const { prefs } = require("sdk/simple-prefs");
+
+const { ToolboxOverlay } = require("./toolbox-overlay");
+
+// Firebug SDK
+const { Dispatcher } = require("firebug.sdk/lib/dispatcher");
+const { Trace/*, TraceError*/ } = require("firebug.sdk/lib/core/trace.js").get(module.id);
+const { System } = require("firebug.sdk/lib/core/system");
+
+// DevTools
+const { ConnectionManager, Connection } = System.devtoolsRequire("devtools/client/connection-manager");
+const { AppManager } = System.devtoolsRequire("devtools/webide/app-manager");
+
+
+const WebIDEConnectionsMonitor =
+/** @lends ConnectionsMonitor */
+{
+  // Initialization
+
+  initialize: function() {
+    Trace.sysout("WebIDEConnectionsMonitor.initialize;", arguments);
+
+    this.onNewConnection = this.onNewConnection.bind(this);
+    this.onDestroyConnection = this.onDestroyConnection.bind(this);
+
+    this.transportObservers = new Map();
+    this.transportListeners = new Map();
```

Comment:
Neither transportObservers nor transportListeners are used, should we remove it?